qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -384,7 +403,12 @@ public void test080ConfigurePasswordThroughEnvironmentVariableFile() throws Exce
         // ELASTIC_PASSWORD_FILE
         Files.writeString(tempDir.resolve(passwordFilename), xpackPassword + "\n");
 
-        Map<String, String> envVars = Map.of("ELASTIC_PASSWORD_FILE", "/run/secrets/" + passwordFilename);
+        Map<String, String> envVars = Map.of(
```

Comment:
Are  users required to do this? Shouldn't auto-configuration be disabled automatically if folks provide a password via this means?

Reply:
No, not at all. I somehow missed removing this disabling of auto-configuration when I changed all DockerTests to work with auto-configured TLS. 

> Shouldn't auto-configuration be disabled automatically if folks provide a password via this means?

We don't disable TLS auto-configuration as it is now, but we won't be attempting to auto-generate and set a password for the elastic user, but respect the what the user confgured.